docs/content/integrations/dbt/reference.mdx

```diff
@@ -82,6 +82,22 @@ The manifest can be created in two ways:
 
 When deploying your Dagster project to production, **we recommend generating the manifest at build time** to avoid the overhead of recompiling your dbt project every time your Dagster code is executed. A `manifest.json` should be precompiled and included in the Python package for your Dagster code.
 
+Creating the manifest at runtime in production is known to cause issues and is not recommended. This is often caused by the following code and should be avoided.
+
+```python startafter=start_troubleshooting_dbt_manifest endbefore=end_troubleshooting_dbt_manifest file=/integrations/dbt/dbt.py dedent=4
+import os
```

Comment:
To make this explicit?

```suggestion
```python startafter=start_troubleshooting_dbt_manifest endbefore=end_troubleshooting_dbt_manifest file=/integrations/dbt/dbt.py dedent=4
"""❌ This is not recommended."""
import os
```

And this for the second code snippet there should be a 

```python
"""✅ This is recommended!"""
```

Reply:
Sure, all changes done in [6614302](https://github.com/dagster-io/dagster/pull/22326/commits/661430260719fc837c59b5824898ff6f9abf6985)

Reply:
Thanks!